templates/slim/block_image.html.slim

```diff
@@ -1,10 +1,11 @@
-.imageblock(id=@id class=[@style,role]
-    style=[("text-align: #{attr :align}" if attr? :align),("float: #{attr :float}" if attr? :float)].compact.join('; '))
-  .content
-    - if attr? :link
-      a.image href=(attr :link)
+- unless attr(1) == 'background'
```

Comment:
You can write this as `(attr :alt)` instead of `(attr 1)`. The first positional attribute is always mapped to the `alt` attribute for images.